app/projectsmodel.cpp

```diff
@@ -97,6 +100,10 @@ void ProjectModel::addProjectFromPath( QString path )
   if ( projectExistsAt >= 0 )
     mProjectFiles.removeAt( projectExistsAt );
 
+  std::shared_ptr<MerginProject> cachedProject = MerginApi::parseProjectData( path, mCachedName );
```

Comment:
I don't like the idea that ProjectModel needs to know the relative path to mergin metadata file - it should not be necessary - let's remove it